fake_system:
  ros__parameters:
    publish_period_ms: 20
    initial_joint_positions: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    initial_gripper_position: 0.0
    publish_intent: false
